scripts/backend_tests_incomplete_coverage.txt

```diff
@@ -48,3 +46,4 @@ core.domain.user_domain_test
 scripts.install_third_party_libs_test
 core.storage.exploration.gae_models_test
 scripts.docstrings_checker_test
+core.domain.value_generators_domain_test
```

Comment:
Can you explain why this test is being removed from its original place in the file and being re-added at the bottom?